schemas/context/segmentmembership.schema.json

```diff
@@ -46,6 +46,14 @@
             "realized": "Entity is entering the segment",
             "exited": "Entity is exiting the segment"
           }
+        },
+        "xdm:propensity" : {
```

Comment:
@vineetathome Yes, this is exactly an indicator of how strong the association between the user and the segment is. I think how that was determined doesn't matter. For this use case, we were thinking of segmentation that happens as a result of some scoring. I think if we'd want to share more detail than a simple propensity on the segment membership, that may need to live in payload (see #411). What are your thoughts?

Reply:
@harleensahni - is confidence a better indicator of "how strong the association between the user and the segment is". For all other scores, i think that they should be a part of the profile.

Reply:
Hey @harleensahni can you comment on Vineet's last?  I feel he has a good point that this could be better called a confidence estimate than a propensity.  I suppose a propensity score is also effectively a confidence estimate, but that term does seem to connote estimated likelihood of a an event occurring, versus strength of a classification.

Reply:
Confidence estimate makes the use case much more narrow. Propensity is more broad.